resources/sass/style.scss

```diff
@@ -344,15 +344,9 @@ hr.small {
   color: #fff;
 }
 
-#video--view #video--sensor {

```

Comment:
Forgot the comma between, with the comma the preview is visible outside the view.
Need to fix the styling somehow to not have the preview zoomed